content/pandas/concepts/dataframe/terms/resetindex/reset-index.md

```diff
@@ -0,0 +1,165 @@
+---
+Title: '.reset_index()'
+Description: 'Resets the index of a DataFrame to be continuous' 
+Subjects: 
+  - 'Data Science'
+  - 'Pandas'
+Tags:
+  - 'Methods'
+  - 'Pandas'
+  - 'Data Science'
+CatalogContent:
+  - 'learn-python'
+  - 'paths/data-science'
+---
+Sometimes there is a need to modify a DataFrame object to better meet a certain use case. When these modifications are applied they often leave smaller DataFrames with indexes that are messy and not continuous. Resetting the index with **.reset_index()** fixes these issues.
+
+## Syntax
+
+```py
+import pandas as pd
+df.read_csv('some_csv.csv', index_col = 'Account Number')
+df.reset_index()
+```
+
+In this syntax example the `index_col = Account Number` parameter sets the index of the DataFrame to a column named Account Number and then df.reset_index() resets the DataFrame index. 
+
+### `.reset_index()` Parameters
+
+The .reset_index() method provides many parameters which allow for more precision when resetting a DataFrame index. The following parameters will be used in the examples later in this article.
+
+- *drop:* The drop parameter can be set to True or False and is set to False by default. When this parameter is set to True it replaces the previous DataFrame index with the new index provided by .reset_index() otherwise it sets the new index in front of the old index.
+- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes .reset_index() makes to the current instance of the DataFrame otherwise it creates a new DataFrame instance with the changes applied to that DataFrame.
```

Comment:
```suggestion
- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes `.reset_index()` makes to the current instance of the DataFrame otherwise it creates a new DataFrame instance with the changes applied to that DataFrame.
```